packages/website/components/navigation/all-packages-nav-content.js

```diff
@@ -9,6 +9,9 @@ import {
 } from '@atlaskit/navigation-next';
 import { gridSize as gridSizeFn } from '@atlaskit/theme';
 
+/** @jsx jsx */
```

Comment:
What does this comment do?

Reply:
https://emotion.sh/docs/css-prop#jsx-pragma

Reply:
weird but ok